src/Trie.mo

```diff
@@ -409,34 +409,33 @@ module {
   /// operation in various ways, and does not (in general) lose
   /// information; this operation is a simpler, special case.
   public func merge<K, V>(tl:Trie<K, V>, tr:Trie<K, V>, k_eq : (K, K) -> Bool) : Trie<K, V> {
-      let key_eq = equalKey<K>(k_eq);
+      let key_eq = equalKey(k_eq);
       func rec(bitpos : Nat, tl : Trie<K, V>, tr:Trie<K, V>) : Trie<K, V> {
-        func lf(kvs : AssocList<Key<K>, V>) : Trie<K, V> = leaf(kvs, bitpos);
         switch (tl, tr) {
           case (#empty, _) { return tr };
           case (_, #empty) { return tl };
           case (#leaf(l1), #leaf(l2)) {
-            lf(
-              AssocList.disj<Key<K>, V, V, V>(
+            leaf(
+              AssocList.disj(
                 l1.keyvals, l2.keyvals,
                 key_eq,
-                func (x : ?V, y : ?V):V {
+                func (x : ?V, y : ?V) : V {
                   switch (x, y) {
                     case (null, null) { P.unreachable() };
                     case (null, ?v) { v };
                     case (?v, _) { v };
                   }
                 }
               )
-            )
+              , bitpos)
```

Comment:
Don't really mind. Check style guide?

Reply:
I guess I'd put the comma at the end of the previous item,

```
     ...),
     bitpos)
```     
but I don't remember what the guide says...